site/src/sphinx/server-grpc.rst

```diff
@@ -93,6 +93,30 @@ a :api:`GrpcServiceBuilder` and add it to the :api:`ServerBuilder`:
     We bound the :api:`GrpcService` without specifying any path mappings. It is because :api:`GrpcService`
     implements :api:`ServiceWithPathMappings`, which dynamically provides path mappings by itself.
 
+.. _server-grpc-decorator:
+
+Decorating a ``GrpcService``
+----------------------------
+
+Unlike a usual Armeria :api:`Service`, :api:`GrpcService` implements a special interface called
+:api:`ServiceWithPathMappings`. You cannot decorate a :api:`GrpcService` using ``Service.decorate()`` but
```

Comment:
Ditto, `cannot` is a bit confusing since it is actually possible, but only when using an explicit path mapping.